### ISSUE TITLE
daily job to send edu-benefits report email

### ISSUE THREAD

**Issue by lihanli** [department-of-veterans-affairs/vets-api#227]

**Comment by jcmeloni** [comment]
@akainic Is this part of the Veterans Day requirements? Thanks!

**Comment by akainic** [comment]
@jcmeloni yes it is! Needs to be done before education form launches

**Comment by akainic** [comment]
Closing, duplicate of #237
